core/contracts/oracle/interfaces/StoreInterface.sol

```diff
@@ -8,30 +8,38 @@ import "../../common/implementation/FixedPoint.sol";
  * @title Interface that allows derivative contracts to pay oracle fees for their use of the system.
  */
 interface StoreInterface {
-    /** 
-     * @dev Pays Oracle fees in ETH to the store. To be used by contracts whose margin currency is ETH.
+    /**
```

Comment:
Same here as Registry -- could you double check that the changes you made in the `Store` comments are reflected in this interface contract? I'm guessing the comments are identical.